docs/integrations/sources/oracle.md

```diff
@@ -132,6 +132,7 @@ Airbite has the ability to connect to the Oracle source with 3 network connectiv
 
 | Version | Date | Pull Request | Subject                                         |
 |:--------| :--- | :--- |:------------------------------------------------|
+| 0.3.16  | 2022-06-22 | [13997](https://github.com/airbytehq/airbyte/pull/13997) | Fixed tests |
```

Comment:
@etsybaev, we don't need a new version for test fixes. Have you already published a new version for this connector? If not, can you delete this version in the changelog?